src/vault.py

```diff
@@ -39,6 +39,24 @@ def check_ip(ip: str):
         sys.exit(1)
 
 
+def check_ip_range(ip_range: int):
+    """
+    Check whether the input IP range is valid or not
+    """
+    try:
+        number = int(ip_range)
+    except ValueError:
+        colors.error('Please enter a valid number for the IP range')
+        LOGGER.error('[-] Please enter a valid number for the IP range')
+        sys.exit(1)
+    else:
+        if number > 0 and number < 255:
```

Comment:
`ip_start_range` < `ip_end_range`, this condition is not checked anywhere.

Reply:
Oh! I forget that. I will fix it

Reply:
I think that this problem is also solved now